worlds/animal_well/region_scripts.py

```diff
@@ -198,11 +198,17 @@ def create_regions_and_set_rules(world: "AnimalWellWorld") -> None:
                 if data.event:
                     location = AWLocation(player, destination_name, None, aw_regions[origin_name])
                     location.place_locked_item(AWItem(data.event, ItemClassification.progression, None, player))
+                elif data.victory:
+                    if options.goal != data.victory:
```

Comment:
```suggestion
                    if data.victory != options.goal:
```
I don't really have a specific thing to reference here, I just don't super like it when comparing an instance of something against a fixed thing, to have the fixed thing on the left side of the comparison.

Like, `if apple_color == color.red` vs `if color.red == apple_color`

so it's fine but also please change it lol

Reply:
100% agreed, and that's how I intuitively write these things. I think I just got lost in the variable sauce a bit here